### DISCUSSION TITLE
建议：增加【勾选隐去已看“watched”的选项】

### DISCUSSION THREAD

**Discussion by My-Favorite-Alpha** [the1812/Bilibili-Evolved#2233] [功能建议 / Ideas]
**建议**：在放映厅（电影、纪录片、电视剧、综艺）、番剧和国创的索引页面，增加如豆瓣电影的【勾选隐去已看“watched”的选项】

**版本**v2.0.0 Technical Preview 7

**浏览器版本**Microsoft Edge(Chromium) 94.0.992.4 (官方内部版本) dev (64 位)

PS：我是GitHub新手，感谢作者开发了这么好用的一款工具，正在使用该版本(v2.0.0 Technical Preview 7)，这个建议也向B站官方发了，但没有收到回复，看了Issues 使用指南和discussion 讨论区使用指南，就想在讨论帖下发条建议。如果作者大大认为这条建议出现过或者不妥就可以*close*或者*我就删去*。

![image](https://user-images.githubusercontent.com/66903838/131332417-97873ba2-51cb-479b-9659-3bf10cb60ad6.png)

![image](https://user-images.githubusercontent.com/66903838/131332431-b20330e4-a3ea-445d-8ddb-15f88eed4ea2.png)

**Comment by the1812** [comment]
怎么才算已看过呢, b站这个页面似乎并没有记录你是否看过

**Reply by My-Favorite-Alpha** [reply]
谢谢回复，我考虑
豆瓣是把视频标记为”看过“，然后在索引页面点击该按钮后就可以过滤掉这个项目；
B站这个页面看不到是否看过，不过是有“追剧”可以标记为“想看、未看和已看”，通过查询“已看”应该可以过滤。

**Reply by the1812** [reply]
可以试试看 (等 2.0 正式发布后
